### ISSUE TITLE
Error: getting package list while browsing for 'Package/Action' in Trace dialogue

### ISSUE THREAD

**Issue by Leucotheia** [google/gapid#315]
I am seeing an error when I try to browse for a Package/Action in the Trace dialogue, the full text of the error is as follows:

The gapit command failed:
23:21:34.063 F: <gapi> Main failed
Error getting package list
Cause: Opening gapid.apk
Cause: open C:\Users\me\gapid\release\pkg\gapid-aarch64.apk: The system cannot find the file specified.

At this point I have already found my device in the Device drop down.

I have not used the application before so it is entirely possible that I have missed something.

**Comment by ben-clayton** [comment]
Pascal - is this the same issue as what Tom hit?

**Comment by pmuetschard** [comment]
I don't know what issue Tom had, but the obvious first question is, does C:\Users\me\gapid\release\pkg\gapid-aarch64.apk exist?

**Comment by Leucotheia** [comment]
Sorry, important information that I left out. No, it doesn't exist. Also, i'm not sure what the path to gapid.apk is but I can't find it anywhere either.

**Comment by pmuetschard** [comment]
The path it's looking for looks correct. The APKs should be built as part of "do build". Can you try running "do build" and verify that it completes successfully? How were you launching the application? "do run gapic"?

**Comment by Leucotheia** [comment]
I was building using do build and running using do run gapic. Initially I had tried a release build, and building again using do build simply told me that everything was up to date. I have just tried building debug rather than trying to delete any intermediate files to force  a rebuild and risk inadvertently deleting something important. The debug build completed successfully, the required apk's are now present and I am able to capture a trace just fine.
